modules/virt-about-upgrading-virt.adoc

```diff
@@ -37,3 +37,13 @@ used to manage the virtual machine process.
 
 * DataVolumes and their associated PersistentVolumeClaims are preserved during
 upgrade.
++
+[IMPORTANT]
+====
+Running virtual machines that cannot be live migrated might block an {product-title} cluster upgrade.
+This includes virtual machines that use hostpath-provisioner storage or SR-IOV network interfaces.
```

Comment:
s/hostpath-provisioner/hostpath provisioner

(to align with the rest of our docs)

Reply:
Done